tests/spread/core22/parallel-install/task.yaml

```diff
@@ -0,0 +1,48 @@
+summary: Verify snapcraft can be installed in parallel
+
+prepare: |
+  #shellcheck source=tests/spread/tools/prepare.sh
+  . "$TOOLS_DIR/prepare.sh"
+
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  set_base "snap/snapcraft.yaml"
+
+  # remove lxd because it does not work after enabling parallel instances
+  # without rebooting the host
+  snap remove lxd --purge
+  snap set system experimental.parallel-instances=true
+
+  install_lxd
+
+restore: |
+  #shellcheck source=tests/spread/tools/prepare.sh
+  . "$TOOLS_DIR/prepare.sh"
+
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  restore_yaml "snap/snapcraft.yaml"
+
+  # remove the parallel installed snapcraft and lxd
+  snap remove snapcraft_a --purge
+  snap remove lxd --purge
+
+  # unset parallel instance support
+  snap set system experimental.parallel-instances=null
+
+  install_lxd
+  install_snapcraft
+
+execute: |
+  #shellcheck source=tests/spread/tools/prepare.sh
+  . "$TOOLS_DIR/prepare.sh"
```

Comment:
isn't this being called in `prepare` already?

Reply:
Yes, but it has to get re-sourced in every section where I need to call a function from that file.